include/ebpf_proxy.h

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "ebpf_protocol.h"
+#include "ebpf_result.h"
+#include "ebpf_structs.h"
+#include "ebpf_windows.h"
+#include "framework.h"
+
+/// @brief Proxy to driver dispatch table.
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    /**
+     * @brief Initialize the eBPF core execution context.
+     *
+     * @retval EBPF_SUCCESS The operation was successful.
+     * @retval EBPF_NO_MEMORY Unable to allocate resources for this
+     *  operation.
+     */
+    _Must_inspect_result_ ebpf_result_t typedef (*ebpf_core_initiate_t)();
```

Comment:
shouldn't typedef be earlier in this line, before the ebpf_result_t return value, like line 49 has
```suggestion
    typedef _Must_inspect_result_ ebpf_result_t (*ebpf_core_initiate_t)();
```